src/Entity/Models/LoanOfficer.cs

```diff
@@ -12,12 +12,12 @@ namespace Entity.Models
     public class LoanOfficer
     {
         [Column("LoanOfficerId")]
-        public Guid LoanOfficerId { get; set; }
+        public Guid Id { get; set; }
```

Comment:
Why are you renaming the database column from `Id` to `LoanOfficerId`? Is there any benefit to doing it this way?